modules/virt-disable-individual-bootsource-update.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-disable-individual-bootsource-update_{context}"]
+= Disabling an automatic update for a custom boot source
```

Comment:
This is the opposite: This method is how to disable ***COMMON*** boot source.

***CUSTOM*** resources are disabled by just removing their object from this list (the HyperConverged's `spec.dataImportCronTemplates` field).

Reply:
Hi @nunnatsa - By "common boot source" do you mean a pre-defined boot source that has not been customized (like RHEL7 for example)? By "custom boot source" I mean an ready-to-use ISO that has been modified by the user in some way. Just need to confirm before I make changes. Thanks.

I don't see we are talking about disabling _custom resources_ on this page... am I missing something that should be documented?

@apinnick - I probably should take a high-level scan at the boot sources doc to see if we can speak to the user in terms of customized and non-customized, rather than introducing jargon that can be misinterpreted. If you want to create a task for next release, I can take that on.

Reply:
Changed this content to reflect that we are speaking of boot sources that have NOT been customized. See new PR: https://github.com/openshift/openshift-docs/pull/48595